framework/src/problems/SubProblem.C

```diff
@@ -1073,6 +1075,13 @@ void
 SubProblem::timestepSetup()
 {
   for (auto & map : _functors)
+    for (auto & pr : map)
+    {
+      std::get<1>(pr.second)->timestepSetup();
+      std::get<2>(pr.second)->timestepSetup();
```

Comment:
umm why do we need to do that?
shouldnt we be wrapping the same functor twice, AD and nonAD wrappers but still same object to set up?